sysbuild/Kconfig.mcuboot

```diff
@@ -175,6 +175,13 @@ config BOOT_SHARED_CRYPTO_ECDSA_P256
 	depends on BOOT_SIGNATURE_TYPE_ECDSA_P256 && !SOC_SERIES_NRF54LX
 	default y
 
+config MCUBOOT_GENERATE_DEFAULT_KMU_KEYFILE
+	bool "Generate default keyfile for provisioning during build"
+	depends on SOC_SERIES_NRF54LX
```

Comment:
nowadays make sense only for nRF54l15/05/10 devices.

Reply:
yes, but later it will be enabled also for nrf54lm20pdk, so maybe better to add all series here, to have that feature ready, to not have to update all that files after enabling KMU in new DKs

Reply:
discussed F2F